website/app/styles/pages/application/sidebar.scss

```diff
@@ -52,6 +52,18 @@ body.application.index {
   }
 }
 
+// ERROR PAGE
+// now with extra specificity
+// also if you want to change this, change footer.scss too
+body.application.error {
```

Comment:
@MelSumner OK, I was sure it worked when I tested the suggested code:
```
body.application.error {
  @include breakpoint.with-fixed-sidebar () {
    .doc-page-sidebar {
      display: none;
    }
  }
}
```
and now I see where the issue is. 

It's not a problem of specificity:
<img width="825" alt="screenshot_2288" src="https://user-images.githubusercontent.com/686239/214161742-e8cce52b-2613-4b71-8ae3-0567a16c6f8e.png">
but of order: the declaration for the `Error` page needs to come *after* the more "generic" declaration for the `not:(.index)` pages.

If you change the order, it will work without need to change specificity.

Reply:
So I guess that has a tradeoff, yeah? Like, do we want the exceptions at the top (maybe easier to find?) then followed by the other things, or the exceptions at the bottom and we all just learn that the exceptions go at the bottom? 

I don't think I have an opinion (it felt like the exceptions were already first so that's where I put them), as long as we are consistent with our own selves. It's one of those things that just gets out of hand really quickly otherwise.

Reply:
I don't think there's a pattern or convention to follow, apart from the way CSS works.
In this case, since we want to override a rule, adding if after makes more sense (and doesn't require specificity battles).